src/core/templates/notify.hbs

```diff
@@ -41,7 +41,7 @@
             </div>
         </div>
         {{! toolbar moved for AA purposes - last tabbable element}}
-        {{#if _isCancellable}}
+        {{#if _showCloseButton}}
```

Comment:
This needs to be wrapping in both
```hbs
{{#if _isCancellable }}
```
and
```hbs
{{#if _showCloseButton}}
```
as the close button is "cancel" behaviour which is blocked by ```_isCancellable:false```, so you could end up with an unusable button by having
```json
{
  "_isCancellable": false,
  "_showCloseButton": true
}
```